mk/container.mk

```diff
@@ -3,17 +3,17 @@ PHONY: build-run clean logs stop help
 .DEFAULT_GOAL:= help
 
 logs: ## View logs of the all the containers
-	docker-compose -f ../dev/docker-compose.yaml logs --follow
+	docker compose -f ../dev/docker-compose.yaml logs --follow
```

Comment:
Why this change?

Reply:
TIL: Looks like "docker compose" is the new way approach and written in GO as compared to `docker-composer`. 

https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command

cc @astronomer/airflow-providers - We should update it in astronomer-providers repo too :)

Reply:
Yes, exactly. It is much nicer also in terms of output. I believe the v1 docker-compose did not have coloured output and is not collapsible.